src/theme/default-theme.js

```diff
@@ -0,0 +1,21 @@
+export default {
+  colors: {
+    primary: '#ebe5da',
+    secondary: '#f04d21',
+    tertiary: '#1e2852'
+  },
+  fonts: {
+    header: 'Helvetica Neue',
+    text: 'Helvetica Neue'
+  },
+  fontSizes: {
+    header: '48px',
+    subHeader: '32px',
+    text: '18px'
+  },
+  space: {
+    headerMargin: '16px',
+    textMargin: '16px',
+    listMargin: '16px 0'
+  }
+};
```

Comment:
These are some defaults for our theme's space, colors, and fonts. https://styled-system.com/getting-started

Reply:
Do you think we should make our theme public/available to the public?

Reply:
Yes! I plan on creating a helper that lets you use or extend it, which really is Object.assign. But still a helper.